### PR TITLE
Yoda topic

### PR DESCRIPTION
**Pull request by peer35**
Yoda topic page based on the text on https://yoda.vu.nl 
#92 #252

### CODE REVIEW

**Review by jhrudey — APPROVED** [review]
Looks good. My only thought is whether its a bit too long or whether there is a way to utilize the iFrame concept to copy over an existing section from the yoda.vu.nl page so that we don't have two separate pages to keep up to date. In other words, perhaps have a brief intro on this page then immediate iFrame to this page https://yoda.vu.nl/site/ that provides the same overview. And finish with the same references at the end. Just my thoughts.

**Comment by peer35** [comment]
Agreed that repeating the same content is not great, but the plan is that the UU Yoda site will replace yoda.vu.nl next year. That means this topic would be the main place for a VU-centered description with a link to the UU Yoda site for more information.

Once the UU site is finished there will probably be some VU-specific Yoda content left that doesn't fit in the UU site. We haven't decided yet where that should go, but I don't want to maintain the Yoda github site just for a few pages. Maybe it will fit a "Yoda guide" in the handbook.

**Comment by jhrudey** [comment]
Ok! In that case, ignore my previous comment. Makes sense not to have the separate github site just for that

Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: Peter Vos ***@***.***>
Verzonden: Wednesday, November 20, 2024 12:01:54 PM
Aan: ubvu/open-handbook ***@***.***>
CC: Hrudey, E.J. (Jessica) ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [ubvu/open-handbook] Yoda topic (PR #270)


Agreed that repeating the same content is not great, but the plan is that the UU Yoda site will replace yoda.vu.nl next year. That means this topic would be the main place for a VU-centered description with a link to the UU Yoda site for more information.

Once the UU site is finished there will probably be some VU-specific Yoda content left that doesn't fit in the UU site. We haven't decided yet where that should go, but I don't want to maintain the Yoda github site just for a few pages. Maybe it will fit a "Yoda guide" in the handbook.

**Review by Karvovskaya — APPROVED** [review]

File: topics/yoda.qmd [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Yoda for the researcher
+Yoda makes use of [iRODS](https://irods.org) as a foundation technology. However, it is much easier to use thereby making it more accessible to a wide range of researchers.
+
+Yoda presents researchers a comfortable, easy-to-use solution for the safe storage, sharing and organisation of research data that follows internationally accepted [FAIR](./fair-principles.qmd) data principles. Something increasingly demanded by both their institutions and funding organisations (such as the NWO).
+
+The following features make Yoda particularly attractive for researchers (end-users):
```
Comment (Jolien-S) [inline topics/yoda.qmd:14]: Should we add the advantage of being able to invite both VU-internal and external colleagues to a Yoda project space? I guess it's kind of implied in the final bullet point, but I think it's useful for researchers to have this explicit.

**Review by Jolien-S — CHANGES_REQUESTED** [review]
Like Peter explained, I think it's useful to have some basic information about Yoda in the handbook. I requested a few changes, after which this topic can be included in the handbook.

**Pull request merged**